assets/js/base/components/filter-element-checkbox/style.scss

```diff
@@ -0,0 +1,38 @@
+.wc-filter-element-checkbox {
+	display: inline-block;
+	position: relative;
+	cursor: pointer;
+	--checkbox-background-color: #{$studio-woocommerce-purple-30};
```

Comment:
Should we move this to the top of this scope?

Reply:
Also, I only see this var is used in one place, so I'm wondering if we can use the `$studio-woocommerce-purple-30` directly?

Reply:
Yes you're right. In previous iterations of this PR I used it in multiple places.